dask/array/fft.py

```diff
@@ -80,6 +99,36 @@ def _ihfft_out_chunks(a, n, axis):
                   'ihfft': _ihfft_out_chunks}
 
 
+def _validate_scipy_fft_func(fft_func):
+    if scipy is not None:
+        if fft_func is scipy.fftpack.rfft:
+            raise ValueError("SciPy's `rfft` doesn't match the NumPy API.")
+        elif fft_func is scipy.fftpack.irfft:
+            raise ValueError("SciPy's `irfft` doesn't match the NumPy API.")
+
+
+def _dim_from_kind(kind):
+    if kind.endswith("n"):
+        return "N"
+    elif kind.endswith("2"):
+        return "2"
+    else:
+        return "1"
+
+
+def _gen_docstring_fft_func(fft_func):
+    func_mod = inspect.getmodule(fft_func)
+    func_name = fft_func.__name__
+    func_fullname = func_mod.__name__ + "." + func_name
+
+    docstring = ""
+    if fft_func.__doc__ is not None:
+        docstring += (fft_preamble % (2 * (func_fullname,)))
+        docstring += fft_func.__doc__
+
+    return docstring
```

Comment:
The reason I was pulling them out is the `fft_wrap` function is exceeding one page in an IDE or terminal. So was thinking it would improve readability to pull out a few smaller functions even if they are only used once.